backend/infrahub/graphql/mutations/ipam.py

```diff
@@ -106,12 +108,14 @@ async def mutate_create(
         ip_address = ipaddress.ip_interface(data["address"]["value"])
         namespace_id = await validate_namespace(db=db, data=data)
 
-        async with db.start_transaction() as dbt:
-            address = await cls.mutate_create_object(data=data, db=dbt, branch=branch)
-            reconciler = IpamReconciler(db=dbt, branch=branch)
-            reconciled_address = await reconciler.reconcile(
-                ip_value=ip_address, namespace=namespace_id, node_uuid=address.get_id()
-            )
+        lock_name = build_object_lock_name(cls._meta.schema.kind)
```

Comment:
The lock for IP address and IP Prefix shouldn't use the kind of the object because we can have multiple kinds that coexist at the same time.
For IP Address and Prefix, we should construct the name of the lock based on the builtin generic `BuiltinIPAddress` and `BuiltinIPPrefix` AND the namespace id